process/incident_management.md

```diff
@@ -0,0 +1,266 @@
+
+#### Responsibilities
+The technical lead of an incident is in charge of leading the technical investigation of the incident and make sure that it is recovered as quickly as possible.
+
+If the incident is not resolved at the end of the current shift then the Incident Tech Lead role gets handed over to the next shift, while Incident Owner is not.
+
+The Incident Tech Lead is responsible to contribute to the documentation of what happened in the RCA document and help the Incident Owner (if not self) to close any gaps in the overall management of the incident
+
+Summary:
+* Is in charge of the investigation and recovery
+* Role is handed over from one shift to the other until full resolution
+* Helps the Incident Owner document the RCA and PMR Action Items.
+
+### Supporting SRE / Parallel Investigator
+A parallel investigator is someone who will be involved hands-on in investigating and helping the fast recovery of the incident.
+A parallel investigator will get involved either by the Incident Commander or by any other members of the IM channel.
+
+#### Responsibilities
+The investigators have the responsibility to describe the action they are taking in the dedicated IM channel to ensure that there is awareness on the steps being taken for the recovery of everyone involved in the process. This can also be done by someone else assisting in the investigation while the main investigator is busy with the recovery.
+
+### Incident Commander
+
+The key element for this role is to ensure that the Incident Commander is solely focused on "oiling the machinery" of incident recovery and will not participate in the hands-on resolution. In Fact the main goal for the Incident Commander is to deal with the overall coordination overhead of the incident recovery, with a cool head, while the investigators are focused 100% on performing the recovery. This will allow faster recovery, better communication flows and less errors during the stress of the incident management.
+
+The Incident Commander can elect someone to be the "scribe" during the incident. The duty of the scribe is to report in the RCA document all the important information (technical details, timeline, ...) while the Incident Commander is focused on helping with the coordination.
+
+#### Responsibilities
+The coordination work under the responsibility of the Incident Commander is as follows:
+1. If the incident has been uncovered by SRE and doesn't have an existing "external" ticket then the Incident Commander will create a ticket to track the overall life of the incident.
+2. Right after the Incident Commander will create a dedicated IM channel:
+  * Making sure that we have a dedicated IM channel opened specifically for the ongoing incident
+    1. Channel is public
+    2. Naming convention for the IM channel should start with the ticket number and the clusterid if available.
+       * Examples:
+         * #inc1096029-<CLUSTERID>
+         * #OHSS-1234-title
```

Comment:
Wonder if examples should be generalized–away from how we do stuff in SD.

Reply:
updated in [513046f](https://github.com/operate-first/sre/pull/17/commits/513046f1ebfb3e49d76d4148a9b94343276d1563)